### PR TITLE
Add GitHub pypi action

### PR DESCRIPTION
**Pull request by ytonui**
adds a github action to push package to pypi

### CODE REVIEW

**Review by nachorpaez — APPROVED** [review]

**Review by piax93 — CHANGES_REQUESTED** [review]
Can we set up also check on PRs like travis used to do? I guess we could have this as examples https://github.com/Yelp/swagger-spec-compatibility/tree/master/.github/workflows and replicate the Travis setup.

Also, I guess the travis config can be removed.

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      run: python -m pip install tox
+
+    - name: Run pre-commit tox job
+      run: tox -e pre-commit
+
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
```
Comment (piax93) [inline .github/workflows/build.yaml:27]: I'd say just ubuntu is enough

File: .github/workflows/build.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        architecture: x64
+
+    - name: Install dependencies
+      run: python -m pip install codecov tox
+
+    - name: Run ${{ matrix.python }} tox job
+      run: tox -e py
+
+    - name: Upload coverage to Codecov
+      run: codecov
```
Comment (piax93) [inline .github/workflows/build.yaml:48]: We don't have this setup

File: .github/workflows/publish.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+    - name: Install Python dependencies
+      run: pip install wheel
+
+    - name: Create a Wheel file and source distribution
+      run: python setup.py sdist bdist_wheel
+
+    - name: Publish distribution package to PyPI
+      uses: pypa/gh-action-pypi-publish@v1.2.2
+      with:
```
Comment (piax93) [inline .github/workflows/publish.yaml:29]: I suppose we'd also need to set `user: yelplabs`

**Review by piax93 — CHANGES_REQUESTED** [review]

**Review by brudman — APPROVED** [review]

**Review by piax93 — APPROVED** [review]

**Pull request merged**